ibis/backends/bigquery/tests/test_client.py

```diff
@@ -75,7 +75,7 @@ def test_compile_toplevel():
 
     # it works!
     expr = t.foo.sum()
-    result = ibis.bigquery.compile(expr)
+    result = ibis.backends.bigquery.compile(expr)
```

Comment:
```suggestion
    result = ibis.bigquery.compile(expr)
```
Quite misleading, I know, but this is the `from ibis.backends import bigquery` in `ibis/__init__.py`, not the module itself.